src/main/java/org/opentripplanner/standalone/config/RouterConfig.java

```diff
@@ -106,6 +110,8 @@ public TransitTuningParameters transitTuningParameters() {
 
     public VectorTileConfig vectorTileLayers() { return vectorTileLayers; }
 
+    public FlexConfig flexConfig() { return flexConfig; }
+
```

Comment:
```suggestion
    public FlexParameters flexParameters(RoutingRequest request) { 
        return flexConfig.toFlexParameters(RoutingRequest request); 
    }

```
The point of this is tp prevent the "Config" class to be part of the public API. This is not 100% clean in the graph-builder, but I would like to more in the right direction.